### DISCUSSION TITLE
Możliwość ustawienia pola publishing->republish przy wystawianiu drafta.

### DISCUSSION THREAD

**Discussion by otwarcie** [allegro/allegro-api#4377] [Suggestions - resolved]
Prosze o umożliwienie ustawiania pola publishing->republish w drafcie oferty (POST /sale/product-offers). Aktualnie pole to można ustawić jedynie późniejszą edycją oferty (PUT /sale/offers/{offerId}). W praktyce oznacza to, że każdą wystawioną ofertę muszę pojedynczo odpytać (pobrać ustawione dane) i następnie wysłać aktualizację (ponownie pełny zestaw danych dla oferty). Czyli zamiast jednego requesta muszę wysłać trzy, a w rzeczywistości: zamiast 5000, wysyłam 15000. 

Dodam, że w API .v1 (POST /sale/offers) było to jak najbardziej możliwe. Parametr 'republish' był przesyłany wraz z draftem oferty i oferta była automatycznie wznawiana po upływie czasu aukcji.

Innym rozwiązaniem jest umożliwienie ustawienia tego pola podczas aktywacji oferty -  PUT /sale/offer-publication-commands/{commandId}. Tym requestem mogę aktywować oferty zbiorczo i gdyby parametr 'republish' był tu akceptowany, to mógłbym zaoszczędzić 10000 requestów dziennie.

**Comment by glorifyingpeace** [comment]
Olbrzymi potrzeba.

**Comment by hahyes** [comment]
Mógłbym to wykorzystać również, dodaje głos. (y)
